src/js/common/utils/extractPoliticianDetailsFromUrl.js

```diff
@@ -0,0 +1,31 @@
+export default function extractPoliticianDetailsFromUrl (url) {
+  // Split the URL into parts using '/'
```

Comment:
Check to make sure url isn't undefined. If it is you will get a crash when trying to split.